in-adapter-api/src/main/java/gohigher/application/ApplicationCommandControllerDocs.java

```diff
@@ -15,11 +18,96 @@
 public interface ApplicationCommandControllerDocs {
 
 	@Operation(summary = "지원서 간편등록")
+	@ApiResponses(value = {
+		@ApiResponse(responseCode = "200", description = "간편 지원서 등록 성공"),
+		@ApiResponse(responseCode = "400", description = "회사명 혹은 직무가 입력되지 않음", content = @Content(
+			examples = {
+				@ExampleObject(name = "회사명 입력되지 않음", value = """
+					{
+					"success": false,
+					"error": {
+						"code": "JOB_INFO_002",
+						"message": "회사명이 입력되지 않았습니다."
+					},
+					"data": null
+					}
+					"""),
+				@ExampleObject(name = "직무 입력되지 않음", value = """
+					{
+					"success": false,
+					"error": {
+						"code": "JOB_INFO_003",
+						"message": "직무가 입력되지 않았습니다."
+					},
+					"data": null
+					}
+					""")
+			}))
+	}
+	)
```

Comment:
```suggestion
	})
```
해당 클래스는 길이가 길어질 수밖에 없을듯한데 조금이라도 줄여보는건 어떨까요ㅠㅠ

Reply:
최대한 닫는 괄호들을 모아서 줄여보았습니다!